WINDOWS.md

```diff
@@ -0,0 +1,58 @@
+## Visual Studio and Ninja Multi-Config Build Instructions
```

Comment:
Let's move this file to be located at `<root>/docs`.  The only document in the root should be the`README.md`, which should redirect to all other documentation (ideally within the docs folder, or subdirectories of that.